packages/designto-flutter/dart-ui/dart-ui-shadow.ts

```diff
@@ -0,0 +1,32 @@
+import * as flutter from "@flutter-builder/flutter";
+import * as dartui from ".";
+import { roundNumber } from "@reflect-ui/uiutils";
+import { TextShadowManifest } from "@reflect-ui/core";
+
+export function shadow(
+  shadows: ReadonlyArray<TextShadowManifest>
```

Comment:
convention not matching. is it correct that Shadow exists under dartui on official flutter docs?

Reply:
https://api.flutter.dev/flutter/dart-ui/Shadow-class.html

Reply:
Alright. still the name and input type does not match intuitively, please add the link - https://api.flutter.dev/flutter/dart-ui/Shadow-class.html to jsdoc for `shadow` func